app/views/carts/show.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <% if @products.empty? %>
+    <div class="col-lg-12 text-center">
+      <%= image_tag "no_cart.png", class: "d-block mx-auto" %>
+      <%= link_to "Continue Shopping", root_path, class: "btn btn-primary mt-3" %>
+    </div>
+  <% else %>
+    <div class="col-lg-4 col-md-12">
+      <h4>Order</h4>
+      <%= form_with url: root_path, method: :post do |form| %>
+        <%= form.label :name, "Name" %>
+        <%= form.text_field :name, class: "form-control" %>
+        
+        <%= form.label :address, "Address" %>
+        <%= form.text_area :address, class: "form-control", rows: 3 %>
+        
+        <%= form.label :phone, "Phone" %>
+        <%= form.text_field :phone, class: "form-control" %>
+        
+        <%= form.submit "Place Order", class: "btn btn-primary mt-3" %>
+      <% end %>
+    </div>
+    <div class="col-lg-8 col-md-12">
+      <div class="row">
+        <div class="col-lg-12">
+          <h4>Cart</h4>
+          <table class="table">
+            <thead>
+              <tr>
+                <th>Product</th>
+                <th class="text-center">Quantity</th>
+                <th>Price</th>
+                <th>Total</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <%= render partial: "product", collection: @products %>
+            </tbody>
+          </table>
+          <p>Total: <%= number_to_currency(@products.sum { |product| product.price * @cart_items[product.id.to_s] }) %></p>
```

Comment:
phần tính toán này e đưa vô helper đi ha

Reply:
Dạ oke a